source/Pacioli/Pacioli.Lib/Models/JournalEntry.cs

```diff
@@ -26,14 +26,19 @@ public JournalEntry(DateTime date, [NotNull] List<JournalEntryLine> debits, [Not
             if (!debits.Any() || !credits.Any())
                 throw new ArgumentException("Debits and credits should contain at least one JournalEntryItem.");
 
+            var creditAccounts = credits.Select(credit => credit.Account);
+            var debitAccounts = debits.Select(debit => debit.Account);
+            if (debitAccounts.Any(debitAccount => creditAccounts.Contains(debitAccount)))
```

Comment:
I think this can be written as:
```
bool hasCommonAccount = debits.Any(debit => credits.Any(credit => debit.Account == credit.Account);
if (hasCommonAccount)
```

Reasoning: No need to create collections for this operation. I created a local test to prove my guess on comparing literals and records. This is just a micro optimization, but if it this is a bit harder to read, please ignore.

Reply:
Thanks, I've put that in!